### PR TITLE
Add .editorconfig

### PR DESCRIPTION
**Pull request by abouthiroppy**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+charset = utf-8
+indent_size = 2
+end_of_line = lf
+indent_style = space
+insert_final_newline = true
```
Comment (icambron) [inline .editorconfig]: Should be false.

**Comment by icambron** [comment]
Looks good other than the one comment. Thanks.

**Comment by abouthiroppy** [comment]
Updated;)

**Comment by icambron** [comment]
Thanks!

**Pull request merged**